### DISCUSSION TITLE
Authorization token

### DISCUSSION THREAD

**Discussion by jlsodai** [pocketbase/pocketbase#1811] [Q&A] [answered]
How do I retrieve/generate Authorization token as an admin or user? Can I use postman for this?

**Comment by ganigeorgiev** [comment, marked as answer]
You can refer to the general "Authentication" guide - https://pocketbase.io/docs/authentication/.

For admins API details, you can check https://pocketbase.io/docs/api-admins/.

For auth records (aka. users) API details, you can check https://pocketbase.io/docs/api-records/#auth-record-actions.
